allennlp/modules/stacked_alternating_lstm.py

```diff
@@ -16,8 +16,10 @@ class StackedAlternatingLstm(torch.nn.Module):
     """
     A stacked LSTM with LSTM layers which alternate between going forwards over
     the sequence and going backwards. This implementation is based on the
-    description in [Deep Semantic Role Labelling - What works and what's next]
-    (https://homes.cs.washington.edu/~luheng/files/acl2017_hllz.pdf).
+    description in [Deep Semantic Role Labelling - What works and what's next][0].
+
+    [0]: https://homes.cs.washington.edu/~luheng/files/acl2017_hllz.pdf
```

Comment:
Can you update this link to point to https://www.aclweb.org/anthology/P17-1044.pdf? The current one is dead.

Reply:
Done, but doesn't it mean that script for url checking is not working (scripts/check_links.py) - masking it is worth making an issue?

Reply:
Hm, that script actually fails when I run it, but due to a different error. If it's something small, I'll just fix it right here.

Reply:
It fails because of other cases of the same problem. I fixed that just now, and added a commit.

Reply:
Now, the `check_links.py` script fails because GitHub is sending out invalid headers. The `urllib` people think they can [wage a Twitter war to get it fixed on the GitHub end](https://github.com/urllib3/urllib3/issues/1822). Best not to get involved.

Reply:
Wow, that's very optimistic I guess.

Reply:
> Now, the `check_links.py` script fails because GitHub is sending out invalid headers. The `urllib` people think they can [wage a Twitter war to get it fixed on the GitHub end](https://github.com/urllib3/urllib3/issues/1822). Best not to get involved.

For what it's worth the person that suggested taking it to Twitter is not a maintainer, and never mentioned a war, just asked for a fix. Looks like it worked according to @andriyor.

Thanks for AllenNLP!

Reply:
Looks like I should give GitHub (and @ssbarnea) more credit :-)

Thanks for urllib!